modin/core/storage_formats/base/query_compiler.py

```diff
@@ -4766,4 +4767,36 @@ def compare(self, other, align_axis, keep_shape, keep_equal, result_names):
             result_names=result_names,
         )
 
+    def repartition(self, axis):
+        """
+        Repartitioning QueryCompiler objects to get ideal partitions inside.
+
+        Allows to improve performance where the query compiler can't improve
+        yet by doing implicit repartitioning.
+
+        Parameters
+        ----------
+        axis : {0, 1}, optional
```

Comment:
```suggestion
        axis : {0, 1, None}, optional
```
I would suggest to explain what happens in each of the cases.